src/ontology/mondo-ingest.Makefile

```diff
@@ -63,8 +63,7 @@ $(TMPDIR)/ordo_relevant_signature.txt: component-download-ordo.owl | $(TMPDIR)
 #  - #60 needs to be fixed at source, but a workaround can probably be implemented in this goal
 $(COMPONENTSDIR)/omim.owl: $(TMPDIR)/omim_relevant_signature.txt | component-download-omim.owl
 	if [ $(COMP) = true ]; then $(ROBOT) remove -i $(TMPDIR)/component-download-omim.owl.owl --select imports \
-		rename --mappings config/property-map-1.sssom.tsv --allow-missing-entities true \
-		rename --mappings config/property-map-2.sssom.tsv --allow-missing-entities true \
+		rename --mappings config/property-map.sssom.tsv --allow-missing-entities true --allow-duplicates true \
```

Comment:
## About running the build
I ran the component and it passes. There should be no difference in any outputs as a result of this code. As such, I would prefer to merge this into develop without running the build. After all, we will always run the build on develop before merging to main, and in the very unlikely event of any issue caused by this, it would be caught at that time.

Reply:
I thought we decided to leave the use of the two property maps as-is to not add a testing burden to improve something that is not broken. 

If this is merged into `develop`, then yes the full build needs to be run.

Reply:
Ah yes, we did discuss that. But circumstances changed a bit.

In #482, the suggested fix for combining these was this, which we decided not to do.
> Merge all mappings into 1 file, and have a second, empty file (to get around ROBOT issue)

While discussing that issue, we also decided that if there is a source that has property mappings not shared by any source, we can have a separate file for that; that's what I did for `icd11.foundation`, so we left that one as-is as well.

But then, Nico asked me to make this issue:
- https://github.com/ontodev/robot/issues/1195

Then during that, we discovered that the proposed flag already exists, and [Nico suggested](https://github.com/ontodev/robot/issues/1195#issuecomment-2066784275) I could potentially test it out to see if it worked.

Reply:
I'll go ahead and run the build for this, then before merging into `develop`.

I don't mean to belabor or offend, but I just wasn't sure if you or Nico considered these additional negative aspects of running builds for all such small PRs, so I want to share:
i. This will discourage me from bothering with otherwise small and quick PRs.
  - In a way that has the benefit that I'll be more focused on bigger, more involved issues.
ii. This is an incentive for me to look for reasons, if possible, to combine small issues into a single PR. But this against the way we do things in Mondo, and a general good practice, to make single PRs for each separate concern, even for small e.g. single line changes.

Reply:
Build success!
Let me know if you want me to upload the build result data files in a separate PR (I think we're doing that step as part of merging into main, and for other large PRs, but i can do it for every PR as well).

Reply:
If you could make a throw-away PR that would help!

Reply:
Here we are!
- #505

Reply:
The data files look good in the other PR.